miio/tests/test_airpurifier.py

```diff
@@ -179,8 +179,3 @@ def test_status_without_temperature(self):
         self.device.state["temp_dec"] = None
 
         assert self.state().temperature is None
-
-    def test_status_string(self):
-        self.device._reset_state()
-
-        assert self.state().__str__
```

Comment:
The status string should be `__repr__` instead of `__str__`, right? https://stackoverflow.com/questions/1436703/difference-between-str-and-repr-in-python